File: app/src/main/java/com/babestudios/companyinfouk/data/model/insolvency/Practitioner.java
package com.babestudios.companyinfouk.data.model.insolvency;


import com.google.gson.annotations.SerializedName;

public class Practitioner {
	@SerializedName("address")
	public Address address;
	@SerializedName("appointed_on")
	public String appointedOn;
	@SerializedName("ceased_to_act_on")
	public String ceasedToActOn;
	@SerializedName("name")
	public String name;
	@SerializedName("role")
	public String role;
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/insolvency/Practitioner.kt
package com.babestudios.companyinfouk.data.model.insolvency


import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Practitioner(
		@SerializedName("address")
		var address: Address? = null,
		@SerializedName("appointed_on")
		var appointedOn: String? = null,
		@SerializedName("ceased_to_act_on")
		var ceasedToActOn: String? = null,
		@SerializedName("name")
		var name: String? = null,
		@SerializedName("role")
		var role: String? = null
) : Parcelable
